### PR TITLE
Added task on cookies (web)

### PR DESCRIPTION
**Pull request by vlppz**

### CODE REVIEW

File: cookiesCTF/main.py [hunk 1]
```diff
@@ -0,0 +1,261 @@
+from bottle import route, run, request, response, static_file, get
+
+
+@route('/')
+def hello():
+    return '''
+	<body>
+		<h1 class=".header1">Select a task or check flag:</h1>
+		<b><a href="/1" class="link">Task №1(Update page)</a>
```
Comment (kolayne) [inline cookiesCTF/main.py:9]:
```suggestion
		<b><a href="/1" class="link">Task №1 (Refresh page)</a>
```

File: cookiesCTF/main.py [hunk 2]
```diff
@@ -0,0 +1,261 @@
+from bottle import route, run, request, response, static_file, get
+
+
+@route('/')
+def hello():
+    return '''
+	<body>
+		<h1 class=".header1">Select a task or check flag:</h1>
```
Comment (kolayne) [inline cookiesCTF/main.py:8]: Did you mean to place a dot here?

File: cookiesCTF/main.py [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+    elif request.get_cookie("is_admin") == "yes":
+        return '''
+		<body>
+			<h1>Hello, Admin!</h1>
+			<p>Flag: PB{eat_my_cookies}</p>
+			<link rel="stylesheet" type="text/css" href="/styles.css">
+			<link rel="preconnect" href="https://fonts.gstatic.com">
+			<link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+			<link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital@1&display=swap" rel="stylesheet">
+		</body>'''
```
Comment (kolayne) [inline cookiesCTF/main.py:85]: What's about `else`? You do not respond if the cookie is not what you expect? I think you should behave in the same way as if the cookie was "no"

File: cookiesCTF/styles.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+.correct {
+	text-align: center;
+	color: #9fff8c;
+}
+
+.try_again {
+	text-align: center;
+	color: #ff7a7a;
+}
```
Comment (kolayne) [inline cookiesCTF/styles.css:43]: Consider adding a new line here

**Review by kolayne — CHANGES_REQUESTED** [review]
Thanks for the PR. I have looked through your code and left some comments. Besides, there are some changes you need to make to it before it is approved:
- In your HTML snippets you have `<link>`s inside of `<body>`s. However, it's suggested that `<link>`s only appear in `<head>`. Fix it, please (you might want to place all your snippets in actual HTML files with proper HTML code (`<html>` tags and so on), put them all in a directory, and use them from your code as static files)
- Place all your files by the following path `/CATEGORY/TOPIC-YEAR[-OTHERINFO]`, for example, `/web/cookies-2021` or `/crypto/base-2021-exam`
- Added a `README.md` file inside your folder with at least a brief explanation of what is in your folder and how to run it (i.e. libraries to install and a command to run your task) - see current directories as examples (they're in Russian, unfortunately).

I will update the main repository's readme to provide information about how contributions should be formatted (directories, readme, ...).

And remember to give commits proper names!

**Comment by kolayne** [comment]
I have added the section for contributors to the repository's README. Please, consider reading it and make sure you have everything requested in your PR

**Comment by vlppz** [comment]
Okay

**Comment by kolayne** [comment]
Please, note that you don't have to create a new Pull Request when you want to update files you're requesting to merge: you can not only push to your branch but also force-push commits if you need to amend something you have already pushed

**Pull request closed without merging**